internal/operands/common-templates/resource.go

```diff
@@ -0,0 +1,136 @@
+package common_templates
+
+import (
+	"bytes"
+	templatev1 "github.com/openshift/api/template/v1"
+	"io"
+	"io/ioutil"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+const (
+	GoldenImagesNSname  = "kubevirt-os-images"
+	bundleDir           = "data/common-templates-bundle/"
+	roleName            = "os-images.kubevirt.io:"
+	Version             = "v0.12.2"
+	kubevirtIo          = "kubevirt.io"
+	commonTemplateLabel = "virt-common-templates"
+)
+
+var commonLabels = map[string]string{
+	kubevirtIo: commonTemplateLabel,
+}
+
+func ClusterRoleName(role string) string {
+	return roleName + role
+}
+
+func readTemplates(filename string) {
+	file, err := ioutil.ReadFile(filename)
+	if err != nil {
+		panic(err)
+	}
+	decoder := yaml.NewYAMLOrJSONDecoder(bytes.NewReader(file), 1024)
+	for {
+		template := templatev1.Template{}
+		err = decoder.Decode(&template)
+		if err == io.EOF {
+			break
+		}
+		if err != nil {
+			panic(err)
+		}
+		if template.Name != "" {
+			templatesBundle = append(templatesBundle, template)
+		}
+	}
+}
+
+func newGoldenImagesNS(namespace string) *core.Namespace {
+	return &core.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+}
+
+func newViewRole(namespace string) *rbac.ClusterRole {
```

Comment:
The `view` role is actually a `Role` and not a `ClusterRole`
https://github.com/kubevirt/kubevirt-ssp-operator/blob/master/roles/KubevirtCommonTemplatesBundle/templates/kubevirt-os-images-viewrole.yaml.j2#L2

Reply:
Corrected